plugins/BEdita/API/tests/TestCase/Controller/ObjectTypesControllerTest.php

```diff
@@ -584,7 +585,34 @@ public function testAddInvalid()
 
         $this->configRequestHeaders('POST', $this->getUserAuthHeader());
         $this->post('/model/object_types', json_encode(compact('data')));
+        $this->assertResponseCode(400);
+        $this->assertContentType('application/vnd.api+json');
+        $this->assertEquals($count, TableRegistry::get('ObjectTypes')->find()->count());
 
+        // add reserved word failure
```

Comment:
I'd prefer to have this test split into its own test case, to better distinguish possible failures.